pyaedt/modeler/circuits/PrimitivesEmit.py

```diff
@@ -743,6 +743,119 @@ def band_stop_frequency(self, band_node, units=""):
             units = self.units["Frequency"]
         return consts.unit_converter(float(band_node.props["StopFrequency"]), "Freq", "Hz", units)
 
+    def set_band_start_frequency(self, band_node, band_start_freq, units=""):
+        """Set start frequency of a given band.
+
+        Parameters
+        ----------
+        band_node : Instance of the band node
+        band_start_freq : float
+        units : str, optional
```

Comment:
```suggestion
        band_node : 
            Instance of the band node
        band_start_freq : float
            Starting frequency of the band node.
        units : str, optional
            Units of the starting frequency. The default is ``""``.
```
Be sure to fill in the missing information. For band_node--what is the data type?  For units, what behavior occurs when the default is used? Phrase it like this: The default is ``""``, in which case X is used.

Reply:
Thanks for the suggestions. I had used band_start_frequency() and band_stop_frequency() as a guide. I will make the changes in a bit and update.